flamingpy/utils/viz.py

```diff
@@ -284,6 +312,149 @@ def draw_EGraph(
     return fig, ax
 
 
+def draw_EGraph_plotly(
+    egraph,
+    color_nodes=False,
+    color_edges=False,
+    label="coordinates",
+    title=None,
+    legend=False,
+    show_axes=True,
+    **kwargs,
+):
+    """Draw the graph state represented by the EGraph with plotly. #TODO: doc
```

Comment:
There are slight differences between the matplotlib and plotly parameters. Should we resolve this with key word arguments? @ilan-tz

Reply:
Then we can indeed move most of the docstring to the main function `draw_EGraph`